src/main/java/org/broadinstitute/hellbender/exceptions/UserException.java

```diff
@@ -457,4 +457,16 @@ public UnimplementedFeature(String message){
             super(message);
         }
     }
+
+    public static final class VQSRPositiveModelFailure extends UserException {
+        private static final long serialVersionUID = 0L;
+
+        public VQSRPositiveModelFailure(String message) { super(message); }
+    }
+
+    public static final class VQSRNegativeModelFailure extends UserException {
+        private static final long serialVersionUID = 0L;
+
+        public VQSRNegativeModelFailure(String message) { super(message); }
+    }
```

Comment:
I'm looking for one of those in the tests now.  I don't like to test for UserException because when the input files are missing, that's a user exception.  (I think Louis and I cleaned that up, but I'm still in favor of being specific.)